src/renderer/redux/sagas/i18n.ts

```diff
@@ -13,8 +13,11 @@ import { all, call } from "redux-saga/effects";
 function* localeWatcher() {
     while (true) {
         const action = yield* takeTyped(i18nActions.setLocale.build);
+
         const translator = diRendererGet("translator");
         translator.setLocale(action.payload.locale);
+
+        document.documentElement.setAttribute("lang", action.payload.locale);
```

Comment:
I do not know if performing DOM mutations from a Redux Saga is "good development practice" (as opposed to doing this from a React component / `*.tsx` file) ... but as far as I am concerned this is an acceptable coding "shortcut", because DOM mutations are hacky in the first place anyway (much like relying extensively on React DOM refs). So, unless somebody has a strong opinion against migrating this minor DOM mutation into renderer-process Redux Saga, I am fine with the proposed change.